src/routes/MemberProfile/MemberProfile.store.js

```diff
@@ -5,6 +5,52 @@ export const FETCH_RECENT_ACTIVITY = 'FETCH_RECENT_ACTIVITY'
 export const FETCH_MEMBER_POSTS = 'FETCH_MEMBER_POSTS'
 export const FETCH_MEMBER_COMMENTS = 'FETCH_MEMBER_COMMENTS'
 export const FETCH_MEMBER_VOTES = 'FETCH_MEMBER_VOTES'
+export const FETCH_PROJECTS = 'FETCH_PROJECTS'
+
+const defaultState = {
+  projects: []
+}
+
+export default function reducer (state = defaultState, action) {
```

Comment:
This is breaking a pattern in `hylo-evo` in that the results aren't getting passed into a ReduxORM model and selected from there, which is what we're doing for everything else. Projects to date were handled as a Post with additional attributes in the ReduxORM modeling. I am all for treating it as a separate type but splitting from ReduxORM is something we should be aware of and discuss I think. 

\* Opinion: I would like to focus the use of component reducers to places where they're only absolutely necessary.

Reply:
ok, after more though I agree that we should stick with the ReduxORM pattern currently in use. @lorenjohnson if you are down to pair with @wushenshen on it that would be much appreciated, though i can also do that

Reply:
moving the projects to the Person type fixes this issue too